creme/sketch/templates/sketch/bricks/chart.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'creme_core/bricks/base/base.html' %}
+{% load i18n creme_core_tags creme_bricks reports_tags %}
```

Comment:
To be cleaned
I presume "reports_tags" is useless
for ""creme_core_tags" use {% load XXX from creme_core_tags %} (these tags are not prefixed)